app/controllers/users_controller.rb

```diff
@@ -1,6 +1,7 @@
 class UsersController < ApplicationController
 
   before_action :set_user, only: [:destroy]
+  #before_action :require_user!
```

Comment:
Should this be enabled really?

Reply:
I expect we want to enable it so that only logged in users can view this controller. I left it commented for the time being because the other controller was broken, so this one was the only way to test.